src/services/basicValidationMapping.js

```diff
@@ -0,0 +1,11 @@
+/** Mapping between basicValidationType and the basic validation login
+ * @param {Object} event - event triggered when a character is typed
+ */
+export const typeToFunctionMap = {
```

Comment:
since this function will be imported in a different file for use, just having `type` would make it very hard to understand the type of what, no?

```suggestion
export const validationTypeToFunctionMap = {
```